archinstall/lib/menu/menu.py

```diff
@@ -158,7 +158,7 @@ def __init__(
 
 		action_info = ''
 		if skip:
-			action_info += str(_("Use ESC to skip"))
+			action_info += str(_("Use ESC to skip\n"))
```

Comment:
Does adding `\n` to the transaction string cause the localization to break? It might need to go outside of the `_()`.

Reply:
`\n` added to **Ctrl+C** functionality too

Reply:
https://github.com/archlinux/archinstall/blob/7943dd82365fd9fb5034a0f1c05de3ccabda468a/archinstall/locales/en/LC_MESSAGES/base.po#L369

I see that this localization exists in English, so this change shouldn't require a new localization line.

Reply:
Ok, then why there is no line space?

Reply:
Apologies for the confusion. I was trying to say that `_("Use ESC to skip\n")` will resolve to the LC_MESSAGES msgid entry. What you have here will work because the `base.po` line I linked exists. The code you have here looks correct.